db/migrate/201509151000_create_qualifying_comments_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateQualifyingCommentsTable < ActiveRecord::Migration
+  def change
+  	create_table :qualifyings do |t|
```

Comment:
I think we should call this "five_a_day_comments" or "qualifying_comments" -- imagine coming to the codebase without any context and seeing "qualifyings". It's pretty confusing.